site/src/components/Docs.svelte

```diff
@@ -486,7 +487,7 @@
         {@html section.metadata.title}
         <small>
           <a
-            href="https://github.com/{owner}/zitadel/edit/master/site/{path}/{dir}/{section.file}"
+            href="https://github.com/{owner}/${project}/edit/master/site/{path}/{dir}/{section.file}"
```

Comment:
this is not resolved in the URI `https://github.com/caos/$zitadel/edit/master/site/docs/administrate/00-overview.en.md`